f5/bigip/cm/trust.py

```diff
@@ -0,0 +1,143 @@
+# coding=utf-8
+#
+# Copyright 2015-2016 F5 Networks Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+
+from f5.bigip.mixins import UnnamedResourceMixin, ExclusiveAttributesMixin
+from f5.bigip.resource import Resource, MissingRequiredCreationParameter, KindTypeMismatch
+
+class Trust(UnnamedResourceMixin, ExclusiveAttributesMixin, Resource):
+    """ Helper class which contains shared methods and attributes of
+        Add_To_Trust and Remove_From_Trust classes.
+
+     .. note::
+         This class inherits from 3 classes due to the requirement of exclusive attributes
+
+    """
+
+    def __init__(self, cm):
+        super(Trust, self).__init__(cm)
+        endpoint = self.__class__.__name__.lower().replace('_', '-')
+        self._meta_data['uri'] = \
+            self._meta_data['container']._meta_data['uri'] + endpoint + '/'
+
+    def update(self, **kwargs):
+        """Update is not supported for trust operations
+
+        :raises: UnsupportedOperation
+        """
+        raise self.UnsupportedMethod(
+            "%s does not support the update method" % self.__class__.__name__)
+
+    def load(self, **kwargs):
+        """Load is not supported for trust operations
+
+        :raises: UnsupportedOperation
+        """
+        raise self.UnsupportedMethod(
+            "%s does not support the update method" % self.__class__.__name__)
+
+    def _kwadd(self, kwargs):
+        """Helper method to append kwargs
+           without unpacking them with "command":"run"
+        """
+        kwargs['command'] = 'run'
```

Comment:
I prefer that we explicitly use:

`kwargs['command'] =  'run'`

instead of a helper method.   In this case we should prefer 'explicit-is-better-than-implicit' in order to make our code more readable.